ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py

```diff
@@ -40,6 +40,8 @@
 from metadata.utils.helpers import get_standard_chart_type, replace_special_with
 from metadata.utils.logger import ingestion_logger
 
+from .client import MetabaseClient
```

Comment:
instead of importing `from .client import MetabaseClient` we should import it like `from metadata.ingestion.source.dashboard.metabase.client import MetabaseClient`